src/app/modules/attack.rs

```diff
@@ -159,4 +159,276 @@ impl AttackData {
         assert_eq!(offset_of!(AttackData, r_fix_damage_speed_up),     0xA4);
         assert_eq!(offset_of!(AttackData, captured_same_time_attack), 0xA6);
     }
-}
\ No newline at end of file
+}
+
+bitflags! {
```

Comment:
I would prefer if all of these enums/constants were in the `app/consts` module, that's how I'm trying to keep things separated. otherwise it looks good to me